dashboards-observability/public/services/saved_objects/event_analytics/saved_objects.ts

```diff
@@ -272,12 +272,11 @@ export default class SavedObjects {
   }
 
   async deleteSavedObjectsList(deleteObjectRequest: any) {
-    const finalParams = {
-      objectIdList: deleteObjectRequest.objectIdList.join(','),
-    };
-    return await this.http.delete(`${OBSERVABILITY_BASE}${EVENT_ANALYTICS}${SAVED_OBJECTS}`, {
-      body: JSON.stringify(finalParams),
-    });
+    return await this.http.delete(
```

Comment:
Just curious: What is the reason behind putting all of the IDs in the parameter instead of sending in the body??

Reply:
for delete with body, it wasn't able to reach to node server in managed service